UnityProject/Assets/Scripts/Items/Implants/Limb.cs

```diff
@@ -34,12 +34,21 @@ public class Limb : BodyPartFunctionality, IMovementEffect
 
 		private PlayerHealthV2 playerHealth;
 
+		public bool IsLeg = false;
```

Comment:
isn't the leg a class? If it is, you can test it for the type instead which is a lot cleaner than adding bools only one type of limb will use

Reply:
A leg isn't a class, legs and arms are limbs and there is no way to tell them apart currently.

Reply:
Is there a leg item trait you can use?

Reply:
how do we store legs stats then? Like movement speed they give

Reply:
hewwo?

Reply:
I think I can allow this as long as we pinky promise to do #8987

Reply:
Pinky promise